.github/workflows/nightly-tests.yaml

```diff
@@ -21,6 +21,8 @@ on:
         description: "The branch to run tests against"
         type: choice
         options:
+          # TODO: undo after testing
+          - rav/multiarch
```

Comment:
?

Reply:
these are just for running nightly tests, I'll undo all updates from this yaml
https://github.com/solo-io/gloo/actions/runs/12142343463/job/33856599541